cli/src/serve.js

```diff
@@ -305,6 +335,8 @@ const read_config_from_file = (project_path, config_file) => {
   for (const field in file_config) {
     if (field === 'connect') {
       parse_connect(file_config.connect, config);
+    } else if (field === 'connect_timeout') {
```

Comment:
You shouldn't have to do this if you have a `connect_timeout` field in the `default_config` object.  It would be a bit more consistent to do it that way.